grml2usb

```diff
@@ -905,14 +905,20 @@ def unmount(target, unmount_options):
             logging.debug("unregister_mountpoint(%s)", target)
             unregister_mountpoint(target)
 
+def extract_device_name(device):
+    match = re.match(r"/dev/([a-z0-9]*[0-9]*)$", device)
+    return match.group(1) if match else None
 
 def check_for_usbdevice(device):
     """Check whether the specified device is a removable USB device
 
     @device: device name, like /dev/sda1 or /dev/sda
     """
 
-    usbdevice = re.match(r"/dev/(.*?)\d*$", device).group(1)
+    usbdevice = extract_device_name(device)
+    if not usbdevice:
+        return 0
```

Comment:
Why did you switch the return value? Return 0 only if the device is a removable USB device should be ok. So IMHO `return 1` would be ok.